openmetadata-docs/content/openmetadata/connectors/database/snowflake/airflow.md

```diff
@@ -14,6 +14,11 @@ If you want to run the Usage Connector, you'll also need to install:
 ```bash
 pip3 install "openmetadata-ingestion[snowflake-usage]"
 ```
+<Note>
+
+For running the usage workflow, the user should be granted the `ACCOUNTADMIN` role (or a role granted IMPORTED PRIVILEGES on the database)
```

Comment:
Maybe we can add a line here specifying that we will be querying the table `ACCOUNT_USAGE` to explain the purpose of the role. Thanks!

Reply:
yes I'll add it there